src/ghci/mod.rs

```diff
@@ -132,21 +129,28 @@ impl Ghci {
         let stderr_handle = task::spawn(async { Ok(()) });
         let stdin_handle = task::spawn(async { Ok(()) });
 
+        let ghci_stdin =
+              GhciStdin {
+                stdin,
+                stdout_sender: stdout_sender.clone(),
+                stderr_sender: stderr_sender.clone(),
+              };
+
         let mut ret = Ghci {
             command: command_arc,
             process: child,
             stdout: stdout_handle,
             stderr: stderr_handle,
             stdin: stdin_handle,
```

Comment:
This handle is no longer associated with a task, so the field should be removed.

Reply:
`ghci_stdin` is also a little bit of a clumsy name (this is the `Ghci` struct, what else would it be `stdin` for?), so maybe we should rename `stdout` and `stderr` to `stdout_handle` and `stderr_handle` (to reflect the fact that they're `JoinHandle`s for tasks) and then `ghci_stdin` can just be `stdin`.